Add adaptation notes (#26)
pxr687/skimage-tutorials-temp: 1_color_images_as_numpy_arrays.Rmd

# What about color?

::: {note}
:class: dropdown
:name: adapted-from-1

> This tutorial is adapted from "`scikit-image`: image processing" by Emmanuelle Gouillart. Please see the References section at the end of the page for other resources that inspired this tutorial.

Reviewer: What about:

```suggestion
This tutorial is adapted from "`scikit-image`: image processing" by Emmanuelle Gouillart. Please see the References section at the end of the page for other sources and resources.
```

?
